Src/BootCamp.Chapter/LINQExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    public static class LINQExtensions
+    {
+        /// <summary>
+        /// Generic Shuffling Algorithm which will accept a collection of type T and return a new shuffled List<T>.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="collection"></param>
+        /// <returns></returns>
+        public static IEnumerable<T> Shuffle<T>(this IEnumerable<T> collection)
```

Comment:
This works, a bit inneficient though. A more efficient way would be to randomize a number, then keep shifting by 1 until you hit a number that wsan't used yet.

Reply:
Ah okay, see I initially did that but my thinking was that for a large dataset that method would be inefficient, I guess I was wrong haha. Thanks for the tip